packages/tradershub/src/screens/PersonalDetails/Sections/AdditionalInformation.tsx

```diff
@@ -38,12 +39,15 @@ const AdditionalInformation = () => {
 
     return (
         <div>
-            <Text as='p' className='my-16' weight='bold'>
-                Additional information
-            </Text>
+            <div className='flex items-center gap-16'>
+                <Text as='p' className='flex-shrink-0 my-16' weight='bold'>
```

Comment:
small nit you dont need to write `flex-shrink-0` you can just write is as `shrink-0`